pypandoc/pandoc_download.py

```diff
@@ -49,9 +49,8 @@ def _get_pandoc_urls(version="latest"):
     # try to open the url
     try:
         response = urlopen(url)
-        content = response.read()
-        pattern = re.compile(r"pandoc\s*([\d.]+)")
-        version = re.search(pattern, content.decode("utf-8")).group(1)
+        version_url_frags = response.url.split("/")
+        version = version_url_frags[7]
```

Comment:
It's sometimes less fragile to get the last element of the array: 

```suggestion
        version = version_url_frags[-1]
```